sky/server/common.py

```diff
@@ -293,7 +293,17 @@ def _start_api_server(deploy: bool = False,
                 time.sleep(0.5)
             else:
                 break
-        logger.info(ux_utils.finishing_message('SkyPilot API server started.'))
+
+        dashboard_msg = (
+            f'{colorama.Style.DIM}Dashboard: {get_server_url(host)}/dashboard.')
+        if not os.path.isdir(server_constants.DASHBOARD_DIR):
+            dashboard_msg += (
+                ' Please run `npm run build` to generate the dashboard if you'
+                ' install SkyPilot from source.')
```

Comment:
```suggestion
                ' Dashboard is not built, run: npm run build')
```

Reply:
It's possible that the dashboard has been built but not the latest, we'd better show a general hint. Or do you think we need to recursively check all the modified files in the dashboard dir to compare with the creation time of `out` to distinguish the `no build` and `non-latest build` case?

Reply:
We can save a version number in the output folder, so we can know what version the previous build was on?

Reply:
I see. How about `Dashboard may be stale when installed from source, to rebuild: npm  --prefix sky/dashboard run build`? Also, make this non-dim and yellow?

Reply:
Updated the message hint with non-dim and yellow:
- If Dev_Version
  - if `out` does not exist
```
✓ SkyPilot API server started. Dashboard: http://127.0.0.1:46580/dashboard
├── Dashboard is not built, to build: npm --prefix sky/dashboard run build
├── SkyPilot API server: http://127.0.0.1:46580
└── View API server logs at: ~/.sky/api_server/server.log
```
  - else
```
✓ SkyPilot API server started. Dashboard: http://127.0.0.1:46580/dashboard
├── Dashboard may be stale when installed from source, to rebuild: npm --prefix sky/dashboard run build
├── SkyPilot API server: http://127.0.0.1:46580
└── View API server logs at: ~/.sky/api_server/server.log
```